packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -959,17 +979,7 @@ - (void)setFlashModeWithResult:(FlutterResult)result mode:(NSString *)modeStr {
 }
 
 - (void)setExposureModeWithResult:(FlutterResult)result mode:(NSString *)modeStr {
-  FLTExposureMode mode = FLTGetFLTExposureModeForString(modeStr);
-  if (mode == FLTExposureModeInvalid) {
-    result(FlutterErrorFromNSError([NSError
-        errorWithDomain:NSCocoaErrorDomain
-                   code:NSURLErrorUnknown
-               userInfo:@{
-                 NSLocalizedDescriptionKey :
-                     [NSString stringWithFormat:@"Unknown exposure mode %@", modeStr]
-               }]));
-    return;
-  }
```

Comment:
I'm now preventing this error from being possible on the Dart side (same below).